File: Teledyne DALSA/Sapera/Demos/NET/GigESeqGrabDemo/VB/GigESeqGrabDemo.vb
Public Class GigESeqGrabDemo
    'The main entry point for the application.
    <STAThread()> _
    Shared Sub Main()
        Application.EnableVisualStyles()
        Application.SetCompatibleTextRenderingDefault(False)
        Dim GigESeq As GigESeqGrabDemodlg = New GigESeqGrabDemodlg()
        If Not GigESeq.IsDisposed Then
            Application.Run(GigESeq)
        End If
    End Sub
End Class

File: Teledyne DALSA/Sapera/Demos/NET/GigESeqGrabDemo/CSharp/GigESeqGrabDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace DALSA.SaperaLT.Demos.NET.CSharp.GigESeqGrabDemo
{
    static class GigESeqGrabDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            GigESeqGrabDemoDlg GigESeq = new GigESeqGrabDemoDlg();
            if (!GigESeq.IsDisposed)
                Application.Run(GigESeq);
        }
    }
}